### PR TITLE
[KAFKA] Migrate from old KafkaSchedulerConfiguration to new ServiceSpec configuration 

### PR DESCRIPTION
**Pull request by mbdcos**

### CODE REVIEW

**Comment by mbdcos** [comment]
retest this please

**Comment by mbdcos** [comment]
retest this please

**Comment by mgummelt** [comment]
I'd prefer if we only had one assigned reviewer on each PR, then cc others you think might be interested.  Otherwise, no one really owns the review.

**Comment by mbdcos** [comment]
This PR touches multiple places in SDK. I opened separate PRs for each and assigned to individuals. Once they are merged, will update this.

File: frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/scheduler/KafkaService.java [hunk 1]
```diff
@@ -1,40 +1,66 @@
...
- * Kafka Service.
+ * Apache Kafka Service
  */
 public class KafkaService extends DefaultService {
     protected static final Logger LOGGER = LoggerFactory.getLogger(KafkaService.class);
 
+
     public KafkaService(File pathToYamlSpecification) throws Exception {
-        super(pathToYamlSpecification);
+        RawServiceSpec rawServiceSpec = YAMLServiceSpecFactory.generateRawSpecFromYAML(pathToYamlSpecification);
```
Comment (mbdcos) [inline frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/scheduler/KafkaService.java]: KafkaService

**Pull request closed without merging**